docs/crashstorage.rst

```diff
@@ -175,7 +175,8 @@ Here is an explanation of each properties of a field:
     Specifies how values are validated when passed to filters of this field
     in Super Search.
 
-    Possible values: ``str``, ``enum``, ``bool``, ``int``, ``datetime``,
+    Possible values: ``str``, ``enum``, ``bool``, ``int``, ``float``,
+    ``datetime``,
```

Comment:
NB: I looked at whether we could merge `query_type` and `data_validation_type` and we might be able to at some point in the future, but there are some fields that have different values.